pages/ksphere/kommander/1.1.0-beta/clusters/index.md

```diff
@@ -10,82 +10,95 @@ excerpt: View clusters created with Kommander or any connected Kubernetes cluste
 
 Kommander allows you to monitor and manage very large numbers of clusters and so we make it easy to either connect existing clusters or create new clusters whose lifecycle is managed by Konvoy.
 
+## Before you begin:
+
+You will already need to have run `konvoy up` with the Kommander addon deployed. You should be looking at the Kommander UI.
+If you are creating a cluster with a cloud partner (Azure, AWS), you will need to configure your [infrastructure provider](/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/).
+If you are planning on attaching more than one cluster, make sure to enter in your valid [license](/ksphere/kommander/1.1.0-beta/licensing/).
+
 From the dashboard page or the clusters page, click Add Cluster and Create Konvoy Cluster.
 
-![Add Cluster Options](/ksphere/kommander/1.1.0-beta/img/add-cluster.png)
+![Add Cluster Options](/ksphere/kommander/1.1.0-beta/img/add-cluster-beta-1-1-0.png)
```

Comment:
'nit: we dont need to have version in filename since the whole img folder already is in its versions folder 👍

Reply:
I realize this - I just named it different so I could keep track when I was making screenshots. I'm indifferent, seems more of an effort to change back now 🤷